oss-training-2019-2.md

```diff
@@ -0,0 +1,13 @@
+# ��Q��׋��� Github�n���Y�I��
+
+
+---
```

Comment:
Add blank line after `---` paginatation.

Reply:
done. sorry, I made one more commit by mistake :p

Reply:
not shown images correctle :(